.github/workflows/integration-test.yml

```diff
@@ -29,11 +37,13 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: Set up Go
-      uses: actions/setup-go@v2
+    - name: Build module
+      uses: ./.github/actions/build-module
       with:
-        go-version: 1.21.13
-    
+        target_os: linux
+        target_arch: ${{ matrix.platform.arch }}
+        docker_image: ${{ matrix.platform.docker_image }}
```

Comment:
you don't still need the go version? does docker already use a specific version so you don't need it here?

Reply:
I think because we're `  using: 'composite'` in the action.yml, the setup go action carries over to the environment of the main workflow